File: docs/_files/registros/Registro.java
import java.util.Scanner;

class Produto    {
    String descricao;
    int lote;
    float peso;
}

public class Registro {

    public static String registroProdutoParaString(Produto prod){
        return String.format("%d:%f:%s", prod.lote, prod.peso, prod.descricao);
    }

    public static void registro() {
        Produto prod = new Produto();
        prod.lote = 10;

        System.out.println(prod.lote);
        // Produto prod_1 = new Produto();
        // Produto prod_2 = new Produto();
        // Produto prod_3 = new Produto();
        // prod_1.descricao = "abc";
        // prod_1.lote = 1;
        // prod_1.peso = 2.0f;

        // System.out.println(prod_1.descricao);
        // System.out.println(registroProdutoParaString(prod_1));

        // prod_2.descricao = "def";
        // prod_2.lote = 1;
        // prod_2.peso = 2.0f;

        // System.out.println(prod_2.descricao);
        // System.out.println(registroProdutoParaString(prod_2));

    }

    public static void registroMain() {
        registro();
    }

    public static void main(String[] args) {
        registroMain();
    }
}

File: docs/_files/registros/Registro.py
from dataclasses import dataclass

@dataclass
class Produto:
  descricao = ""
  lote = 0
  peso = 0.0

def registroProdutoParaString(prod):
    return f"{prod.lote}:{prod.peso}:{prod.descricao}"

#variáveis
prod_1 = Produto()
prod_2 = Produto()
prod_3 = Produto()

#definição dos valores a cada campo
prod_1.descricao = "leite"
prod_1.lote = 100
prod_1.peso = 0.750

prod_2.descricao = "chocolate"
prod_2.lote = 84
prod_3.peso = 0.150

prod_3.descricao = "agua mineral"
prod_3.lote = 502
prod_3.peso = 0.450

print(registroProdutoParaString(prod_1))
print(registroProdutoParaString(prod_2))
print(registroProdutoParaString(prod_3))

